bin/install.sh

```diff
@@ -102,9 +103,8 @@ if [[ "$OS" = "Linux" && $(cat /proc/1/cgroup) == *cpu:/docker/* ]]; then
   # Java Install
   echo -e "\033[1;36mStarting Java install...\033[0m"
 
-  sudo add-apt-repository -y ppa:openjdk-r/ppa
   sudo apt-get update
-  sudo apt-get install openjdk-8-jdk libgfortran3 -y
+  sudo apt-get install openjdk-7-jdk libgfortran3 -y
```

Comment:
@pferrel Do you have a fix for https://github.com/apache/incubator-predictionio/blob/develop/data/src/main/scala/org/apache/predictionio/data/storage/AccessKeys.scala#L69 that uses Java 8 API?